src/components/HURUmap/Panel/MobilePanel/index.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import Panel from ".";
+
+import { panelArgs } from "@/pesayetu/config";
+
+export default {
+  title: "Components/HURUmap/MobileExplore",
```

Comment:
1. What's a `MobileExplore`?
2. Single story on main Panel itself is enough... when one switches screen size, the component will switch to mobile view.